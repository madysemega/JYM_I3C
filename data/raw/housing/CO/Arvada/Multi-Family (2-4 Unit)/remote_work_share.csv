predictions_remote_work_share
4.780422422885894
3.3476252651214597
4.153808903694152
3.184892494678497
2.845236964225769
2.412393707036972
2.78704204082489
2.5436364781856535
2.419995757341385
2.7324046683311463
1.965071197748184
2.1683709979057313
2.8830620718002318
1.7405391693115233
2.363001880645752
2.0094415420293807
2.100270674228668
2.304135661125183
1.9023099875450133
1.2034277182817459
1.1414011114835738
1.6705888992547988
1.6021080350875854
1.5077837634086608
2.1919551515579223
1.4165553772449493
1.6334043300151824
