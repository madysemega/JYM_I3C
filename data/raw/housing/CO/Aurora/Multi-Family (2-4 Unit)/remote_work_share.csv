predictions_remote_work_share
7.464161727428436
7.1889644360542295
7.094764611721039
8.288542506694794
6.814986160993576
7.592821720838547
6.7264636969566345
7.1017976260185245
8.404405484199524
7.861295363903046
8.812179992198944
8.222600934505463
7.24755373954773
7.483859441280365
6.881618038415909
7.355046412944794
6.792163238525391
8.885988030433655
7.222398564815522
7.126204577684403
8.998113188743591
6.870008573532105
8.563238785266876
6.573214051723481
7.26822616815567
7.321432880163193
7.414836452007294
