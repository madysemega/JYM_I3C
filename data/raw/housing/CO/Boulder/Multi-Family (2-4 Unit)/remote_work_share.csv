predictions_remote_work_share
16.341599857211115
16.790159806609157
18.67580521643162
16.883392704725267
16.64124995172024
15.502500675320627
14.287666478753092
16.282203252911568
16.08274296462536
13.90255745649338
13.050065064430239
15.384450156688692
16.09204776465893
15.30096829235554
15.413955297470094
17.50782143175602
16.498250628113748
15.786945767998697
14.378025714159014
14.024312128424645
15.520831379294398
14.197595435976984
14.70319273531437
14.581875000596048
16.28250280797482
16.49548645794392
15.19349000155926
