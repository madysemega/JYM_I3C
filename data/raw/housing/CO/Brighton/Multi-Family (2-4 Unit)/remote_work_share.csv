predictions_remote_work_share
10.80735691666603
6.004549487829208
8.723787694573401
8.60404320001602
5.935669763088226
7.7888813620805735
7.603812080025673
7.132907361388206
4.60931170374155
8.003329846858978
4.165039558410644
8.818723289966583
9.636469171643256
7.665463528633117
9.559896805286407
8.597364769577979
7.394840682148933
9.388912698626518
4.566736781895161
11.194239206314085
6.556781698465347
4.100771722197532
9.8635964512825
10.235836211442948
7.8307556927204125
9.309804226756095
5.360732737183571
