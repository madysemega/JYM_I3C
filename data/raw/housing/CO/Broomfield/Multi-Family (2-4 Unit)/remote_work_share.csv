predictions_remote_work_share
18.725489767789846
11.607584156990052
20.85214416623116
14.812544339299205
12.705875357389452
14.096600090861322
16.9319031560421
19.81506347894669
17.087317343950275
20.079358371496205
8.514786515533926
18.0427032583952
20.15391089558602
8.416716294884683
20.059065266847615
17.223671224117282
15.47835893571377
18.49160386562348
16.38027356207371
12.833920636177064
19.319102945327764
13.12219475746155
15.049749388694766
19.236748840808872
19.27841246962548
14.834809305667878
14.848222872018816
