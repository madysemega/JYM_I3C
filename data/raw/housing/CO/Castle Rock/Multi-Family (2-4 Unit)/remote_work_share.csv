predictions_remote_work_share
4.3992413234710686
4.851400337219238
5.579936189651489
3.8740713644027704
4.247916994094848
3.0904090404510494
3.940058999061584
3.960339422225952
4.23658896446228
3.6569181823730466
3.651008038520813
4.2298287105560295
4.404567799568176
4.049893827438354
5.231539015769958
4.468625335693359
4.101358833312988
3.7102818012237546
3.4351088619232173
3.72938642501831
4.535321226119994
4.66046175956726
4.305596842765808
4.433902182579041
4.148671131134033
4.607226452827454
4.543396763801574
