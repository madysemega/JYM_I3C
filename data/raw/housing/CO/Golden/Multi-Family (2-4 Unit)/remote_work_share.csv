predictions_remote_work_share
9.49855177283287
9.494615132808685
11.721358417272565
10.88380537509918
9.837081921100616
10.237560993432998
6.55447106719017
7.2471390414237975
10.262232898473739
7.401056995987892
5.0002427533268925
8.853435657024383
8.459627856016159
7.971671922802925
9.693497800230979
8.490891144275665
10.753544746637344
8.302197897434235
7.413411998748779
5.181352160871029
8.013402396440506
5.176824041604996
6.400516241192817
7.919425716400147
7.978367775082588
9.758909566402435
10.601493101119994
